README.md

```diff
@@ -32,6 +32,11 @@ config :gandalf, auth_key: "auth_key_goes_here"
 Plug Gandalf, auth_key: "auth_key_goes_here"
 ```
 
+
+### Whitelisting
+Paths:
+To whitelist paths, simply provide `whitelistd_paths: ~r/path1|path2/` to config.
```

Comment:
No section on IP.

`To whitelist paths add... to config` seems somewhat better to me.

Reply:
`whitelistd` <- missig `e` :wink: